src/Lean/Data/RBMap.lean

```diff
@@ -225,6 +225,28 @@ inductive WellFormed (cmp : α → α → Ordering) : RBNode α β → Prop wher
   | insertWff {n n' : RBNode α β} {k : α} {v : β k} : WellFormed cmp n → n' = insert cmp n k v → WellFormed cmp n'
   | eraseWff {n n' : RBNode α β} {k : α} : WellFormed cmp n → n' = erase cmp k n → WellFormed cmp n'
 
+section Map
+
+@[specialize] def mapM {α : Type v} {β γ : α → Type v} {M : Type v → Type v} [Applicative M]
+  (f : (a : α) → β a → M (γ a))
```

Comment:
Why `a` is explicit here, and implicit at `map`? I expected both to be implicit.
Is this a workaround for some kind of Lean bug/limitation?

Reply:
this was a mistake by me